### ISSUE TITLE
How to skip intro videos using controllers?

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [fholger/farcry_vrmod#44]
How to skip intro (Crytek, Ubisoft, etc.) videos when starting a game using controllers only? I've tried different buttons on the controllers, none works for me. I'm forced to use the space bar on my keyboard to do this. IMO, using the Right (and/or Left) hand controller trigger for this seems most expected and handy.

**Comment by FluorescentHallucinogen** [comment]
> but you need to have your controllers pointed onto the video

Is it technically possible to remove this requirement?

**Comment by fholger** [comment]
Yes, but the effort required to do so is not worth it.

**Comment by FluorescentHallucinogen** [comment]
> It is the triggers, but you need to have your controllers pointed onto the video and then pull and hold the trigger. Basically, you need to have a vritual laser pointing towards the video for the skip to work.

Just checked again, it doesn't work too. :(

To clarify: I'm talking specifically about the intro videos right after starting the game, not between levels.

**Comment by fholger** [comment]
So am I, and it is definitely working. Though you can just delete the videos from the game install if you prefer.

**Comment by FluorescentHallucinogen** [comment]
Oh, today I finally managed to get it to work! It's my impatience and haste. It's not enough to just click on the video, you have to hold the controller trigger long enough.

On the keyboard, all it takes is one short key press to skip the video.

Is it possible to reduce the time users have to hold the trigger to skip the video in VR?

**Comment by FluorescentHallucinogen** [comment]
@fholger Is it possible to reduce the time users have to hold the trigger to skip the video in VR? I.e. short click.

**Comment by fholger** [comment]
Yes, but the long press is by design. A short click is too easy to trigger accidentally. Again, if you want to get rid of the intro videos, just delete the video files, much more convenient than any skip function.
